src/utilities/breakpoints.ts

```diff
@@ -10,15 +10,19 @@ const noWindowMatches = {
   addListener: noop,
   removeListener: noop,
   matches: false,
-};
+  onchange: noop,
+  addEventListener: noop,
+  removeEventListener: noop,
+  dispatchEvent: (_: Event) => true,
+} as MediaQueryList;
```

Comment:
By casting this object as an instance of MediaQueryList then the two function below can infer their return type correctly, so we don't need to explictly state it

Reply:
Rather than casting we should type the variable

```tsx
const noWindowMatches: MediaQueryList  = {
```